### ISSUE TITLE
"Create" partition button crashes Calamares when no partition ("free space") is selected

### ISSUE THREAD

**Issue by hexarch** [calamares/calamares#1900]
Calamares 3.2.4 on NetRunner 21.01.0 "XOXO" ISO netrunner-desktop-2101-64bit.iso

Install was done in a VM on QEMU on ArchLinux BIOS/XFS.

**Describe the bug**
Pressing "Create" crashes Calamares when no partition is selected.

Partition config in a 60GB virtual hd:

- erase disk: will create a 51.5gb ext4 + 8.5gb swap (wow!)
- choose manual partitioning (unpartitioned space...)

**To Reproduce**
Create a new partition
- press "new partition table" and choose "mbr"
- select the "free space" line
- press "new partition table" and choose "mbr"
- notice that "free space" line get's unselected
- press "Create" and Calamares crashes, with no error message

**Expected behavior**
Do nothing or warn that no partition is selected.

**Comment by demmm** [comment]
Please check with your used distribution first.  This is referring to a release of Calamares from over three years ago (February 2019).  There have been over 50 releases since, so it is time for your distribution to use an actual supported calamares.

**Comment by hexarch** [comment]
I only noticed that when I checked Netrunner OS kernel and other programs versions, that are really old. This was also an eye open moment for me to stay within Arch, Manjaro and a few more distros, not many, because of their "stable" invalid argument,
It was not only Netrunner OS, but a few more distros that I was checking this Sunday on real hardware.
I'm closing this because just confirmed that this no longer occurs on the latest Manjaro Calamares version. Thank you
